posh-git.psm1

```diff
@@ -92,7 +92,7 @@ if (!$currentPromptDef -or ($currentPromptDef -eq $defaultPromptDef)) {
         if ($GitPromptSettings.DefaultPromptEnableTiming) {
             $sw.Stop()
             $elapsed = $sw.ElapsedMilliseconds
-            Write-Host " $elapsed mS" -NoNewline
+            Write-Host " $($elapsed)ms" -NoNewline
```

Comment:
Works for me.  You can simplify that a bit `" ${elapsed}ms"`.  In interpolated strings in PowerShell, you can delineate variable names with `${<variable-name>}`.